budget/app.py

```diff
@@ -157,7 +123,7 @@ def create_budget(synapse_id, team):
       'TagKeyValue': [
         (
           'aws:servicecatalog:provisioningPrincipalArn$arn:aws:sts::'
-          f'{configuration["account_id"]}:assumed-role/ServiceCatalogEndusers/{synapse_id}'
+          f'{configuration.account_id}:assumed-role/ServiceCatalogEndusers/{synapse_id}'
```

Comment:
Can this filter have wildcards? It seems reasonable to either parametrize the role name or exclude and filter on any role with `/{synapse_id}` appended as a session name. I'm not sure the likelihood of changing role names or of someone having multiple roles.

Reply:
No wild cards. However, thank you for point that out -- I'm going to go ahead and make this configurable or something because I have no data to test with for the role I actually need, `ServiceCatalogExternalEndusers`, and have been using `ServiceCatalogEndusers` instead -- which is an error.